src/cluster.c

```diff
@@ -2296,7 +2315,11 @@ int clusterProcessPacket(clusterLink *link) {
             message = createStringObject(
                         (char*)hdr->data.publish.msg.bulk_data+channel_len,
                         message_len);
-            pubsubPublishMessage(channel,message);
+            if (type == CLUSTERMSG_TYPE_PUBLISHSHARD) {
+                pubsubPublishMessageShard(channel, message);
+            } else {
+                pubsubPublishMessage(channel,message);
```

Comment:
`pubsubPublishMessageShard(channel, message)` parameters have space but `pubsubPublishMessage(channel,message);` don't have space. 

what about this? `pubsubPublishMessage(channel, message);`